nemoguardrails/library/autoalign/flows.co

```diff
@@ -0,0 +1,33 @@
+define subflow autoalign check input
+  $input_result = execute autoalign_input_api(show_autoalign_message=True)
+  if $input_result["guardrails_triggered"]
+    $autoalign_input_response = $input_result['combined_response']
+    bot refuse to respond
+    stop
+
+define subflow autoalign check output
+  $output_result = execute autoalign_output_api(show_autoalign_message=True)
+  if $output_result["guardrails_triggered"]
+    bot refuse to respond
+    stop
+  else
+    $pii_message_output = $output_result["pii_fast"]["response"]
+    if $output_result["pii_fast"]["guarded"]
+      bot respond pii output
+      stop
```

Comment:
This approach does not play well if there are multiple output rails. If this is the first one, it will not allow the others to run. Instead the `$bot_message` context variable should be altered, e.g., `$bot_message = $pii_message_output`.

Reply:
resolved in b49cb444ac134ee061cbae13f72723b66aa0d6bd